lib/cc/analyzer/issue_validator.rb

```diff
@@ -5,6 +5,7 @@ class IssueValidator
         IssueCategoryValidation,
         IssueCheckNamePresenceValidation,
         IssueDescriptionPresenceValidation,
+        IssueLocationExistenceValidation,
```

Comment:
Presence means "is it present in the output?", whereas existence means "does it actually exist where the output says it is?"